librdkafka.yaml

```diff
@@ -51,7 +51,7 @@ subpackages:
     description: librdkafka dev
 
 update:
-  enabled: true
+  enabled: false
```

Comment:
nit: maybe add a comment explaining why this is disabled? to make it easier to follow-up